src/internal/svg/successSvg.tsx

```diff
@@ -11,7 +13,7 @@ export const successSvg = (
     <title>Success SVG</title>
     <path
       d="M8 0C3.58 0 0 3.58 0 8C0 12.42 3.58 16 8 16C12.42 16 16 12.42 16 8C16 3.58 12.42 0 8 0ZM6.72667 11.5333L3.73333 8.54L4.67333 7.6L6.72667 9.65333L11.44 4.94L12.38 5.88L6.72667 11.5333Z"
-      fill="#65A30D"
+      className={className}
```

Comment:
This will break color styling

Reply:
@dschlabach  It used to be hardcoded `65A30D`, and now I'm using `icon.success`. Do you mean that `icon.success` is a different tone than `65A30D` ?

Reply:
This is likely hardcoded because the icon fill color should not update based off of themes / modes